hug/interface.py

```diff
@@ -341,6 +341,9 @@ def _rewrite_params(self, params):
             if interface_name in params:
                 params[internal_name] = params.pop(interface_name)
 
+    def _remap_entry(self, interface_name):
+        return self.map_params.get(interface_name, interface_name)
```

Comment:
Attempt to find mapping for the type. If it is empty, then default is just the original parameter.